src/Core/SpatialObjects/ConvertSpatialObjectToImage/Code.cxx

```diff
@@ -87,16 +87,10 @@ main(int argc, char * argv[])
 
   imageFilter->SetOutsideValue(0);
 
-  using WriterType = itk::ImageFileWriter<ImageType>;
-  WriterType::Pointer writer = WriterType::New();
-
-  writer->SetFileName(argv[1]);
-  writer->SetInput(imageFilter->GetOutput());
-
   try
   {
     imageFilter->Update();
-    writer->Update();
+    itk::WriteImage(imageFilter->GetOutput(), argv[1]);
```

Comment:
I think this is also a label image, so compression is desirable.

Reply:
Done